### PR TITLE
Remove duplicate `large-memory` label

### PR DESCRIPTION
**Pull request by Crola1702**
@Blast545 noticed that this document had duplicated the `large-memory` label. I think the one that remains is the most clear and concise between the two definitions.

### CODE REVIEW

**Review by Blast545 — APPROVED** [review]
Maybe you can combine the two definitions? but that's a nitpick, lgtm. Let's wait for codeowners to comment before merging.

**Comment by j-rivero** [comment]
> Maybe you can combine the two definitions? but that's a nitpick, lgtm. Let's wait for codeowners to comment before merging.

I think that we good with the changes.

**Pull request merged**
